features/providers/sign_in.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Signin in
+
+  @javascript @mock_auth_enabled @vcr_turned_off
```

Comment:
This is duplicating a system test in `spec/system/mock_auth/sign_in_spec.rb` 
Should that be extended rather this duplication?